nanome/api/ui/button.py

```diff
@@ -405,6 +405,16 @@ def set_all(self, value):
             self.selected_highlighted = copy.deepcopy(value)
             self.unusable = copy.deepcopy(value)
 
+        def set_each(self, **values):
```

Comment:
shouldn't this just call `self._set_all(values)`? same with `set_all` above tbh

Reply:
documenting function overloads is...real hard. they do different things so I wanted to keep them separate